storybook/storybook.js

```diff
@@ -109,9 +103,12 @@ configure(() => {
 }, module);
 /* eslint-enable global-require */
 
-const StorybookUI = getStorybookUI({ onDeviceUI: false });
+const StorybookUI = getStorybookUI({
+  onDeviceUI: false,
+  asyncStorage: null,
```

Comment:
See https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#react-native-async-storage.

I tried to get it working but it caused a world of hurt, and for such a small feature I thought it's not worth the hassle. We can revisit this in the future if necessary.